sdk/core/azure-core/inc/azure/core/cryptography/hash.hpp

```diff
@@ -113,18 +113,23 @@ namespace Azure { namespace Core { namespace Cryptography {
 
   public:
     /**
-     * @brief Construct a default instance of @Md5Hash.
+     * @brief Construct a default instance of #Azure::Core::Cryptography::Md5Hash.
      */
     explicit Md5Hash();
 
     /**
-     * @brief Cleanup any state when destroying the instance of @Md5Hash.
+     * @brief Cleanup any state when destroying the instance of #Azure::Core::Cryptography::Md5Hash.
      */
     ~Md5Hash();
 
   private:
     void* m_md5Context;
 
+    // Delete the copy and move constructors, along with the assignment operator.
+    Md5Hash(Md5Hash const&) = delete;
+    Md5Hash(Md5Hash const&&) = delete;
+    void operator=(Md5Hash const&) = delete;
```

Comment:
```suggestion
    Md5Hash(Md5Hash const&) = delete;
    Md5Hash(Md5Hash&&) = delete;
    Md5Hash& operator=(Md5Hash const&) = delete;
    Md5Hash& operator=(Md5Hash&&) = delete;
```

Reply:
See discussion above, which suggests removing one rather than adding one: https://github.com/Azure/azure-sdk-for-cpp/pull/1647#discussion_r573231549